test/unit/uuidToBytes.test.js

```diff
@@ -0,0 +1,46 @@
+import assert from 'assert';
+import uuidv4 from '../../src/v4.js';
+import uuidToBytes from '../../src/uuidToBytes.js';
+import bytesToUuid from '../../src/bytesToUuid.js';
+import gen from 'random-seed';
+
+// Use deterministic PRNG for reproducable tests
+const rand = gen.create('He who wonders discovers that this in itself is wonder.');
+function rng(bytes = []) {
+  for (let i = 0; i < 16; i++) {
+    bytes[i] = rand(256);
+  }
+  return bytes;
+}
+
+describe('uuidToBytes', () => {
+  test('string->bytes->string symmetry', () => {
+    // Verify string->bytes->string symmetry for a selection of uuids
+    for (let i = 0; i < 1000; i++) {
+      const uuid = uuidv4({ rng });
+      assert.equal(bytesToUuid(uuidToBytes(uuid)), uuid);
```

Comment:
Do we test `bytesToUuid` separately? If not, then in principle both methods could be buggy in way where the bugs just compensate each other.

To be on the safe side we should add an additional test case where we _explicitly_ test that `uuidToBytes` for a fixed UUID yields a fixed expected byte array.

Reply:
Added explicit byte array test.